plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -146,7 +146,7 @@ contract RootChain {
         private
     {
         uint256 blknum = utxoPos / 1000000000;
-        uint256 priority = Math.max(childChain[blknum].created_at, block.timestamp - 1 weeks);
+        uint256 priority = Math.max(childChain[blknum].created_at + 2 weeks, block.timestamp + 1 weeks);
```

Comment:
I'd make it somehow clear, that this is the `exitable_at` used later. I'am aware that having two variables here (exitable at and priority) could incur gas cost (or doesn't it? - that's something I'd expect a dumbest compiler to optimize out).

So - if we can check that the optimizer will take gas burden away, I'd go for renaming to `exitable_at` and `priority`.
Otherwise, some comment perhaps, that this is the exitable_at (or scheduled finalization time) of an exit request?